avocado/core/test.py

```diff
@@ -961,15 +957,14 @@ def test(self):
         raise exceptions.NotATestError(e_msg)
 
 
-class SkipTest(Test):
+class FakeTest(Test):
```

Comment:
As for the name, have you considered `MockTest`? `Fake` for me means something like fraud, but `Mock` is more about replacing self instead of something. Anyway I'm not native, so don't know...